TeamCode/src/main/java/org/firstinspires/ftc/teamcode/auto/ColorDetec_Right.java

```diff
@@ -0,0 +1,109 @@
+package org.firstinspires.ftc.teamcode.auto;
```

Comment:
Have a more descriptive file name or comment at the top of this file summarizing the auton